src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotator.java

```diff
@@ -97,7 +104,10 @@
 @BetaFeature
 @DocumentedFeature
 public class VariantAnnotator extends VariantWalker {
+    public static final String EXPRESSION_LONG_NAME = "expression";
```

Comment:
Is the caveat at line 92 that we can't output every annotation still true?

Reply:
no longer true.  Removed.